frontend/src/containers/EditCompanyModal.tsx

```diff
@@ -14,31 +25,35 @@ class EditCompanyModal extends Component {
             modal: false
         };
 
-        this.handleChange = this.handleChange.bind(this);
+        this.handleNameChange = this.handleNameChange.bind(this);
+        this.handleOrgNrChange = this.handleOrgNrChange.bind(this);
+    
         this.toggle = this.toggle.bind(this);
         this.apiUpdateCompany = this.apiUpdateCompany.bind(this);
     }
 
-    handleChange(event) {
-        const name = event.target.name;
-        const value = event.target.value;
-        this.setState({ [ name ]: value });
+    handleNameChange(event: React.ChangeEvent<HTMLInputElement>) {
+        this.setState({ companyName: event.target.value });
+    }
+
+    handleOrgNrChange(event: React.ChangeEvent<HTMLInputElement>) {
+        this.setState({ orgNr: event.target.value });
     }
 
     toggle() {
-        this.setState((prevState) => ({
-            modal: !prevState.modal
-        }));
+        this.setState({
+            modal: !this.state.modal
```

Comment:
her skal man helst bruke prevState for å være sikker på at man har verdien før oppdateringen (referansen kan jo f.eks. endres av en annen metode e.l. fra vi starter denne til operasjonen kjører).
```suggestion
        this.setState(prevState => ({
            modal: !prevState.modal
```